.github/workflows/upload-and-deploy.yaml

```diff
@@ -45,6 +45,10 @@ jobs:
       - name: Copy files to templates bucket
         run: python src/scripts/manage_artifacts/artifacts.py --upload --ref $NAMESPACE
 
+        # Hardcode --ref as stopgap for actual job script versioning
+      - name: Copy Glue scripts to templates bucket
+        run: python src/scripts/manage_artifacts/artifacts.py --upload --ref "v0.1"
```

Comment:
Why is it v0.1 here?  Is this in reference to github release versions?

Reply:
Back when Tess and I were first working on Glue jobs we began using this crude versioning system so that different versions of the S3 to JSON job could potentially be deployed simultaneously. The idea was that older studies would need older implementations whereas newer studies could use the latest and greatest implementation. There hasn't been a real need yet for supporting multiple versions of the S3 to JSON job, but the versioning system has stuck around. The glue jobs have their job scripts store to this "v0.1" namespace, which is separate from where we store other artifacts like schema mapping.

Reply:
Eventually, I'd like to re-design this.